cmd/system-probe/modules/gpu.go

```diff
@@ -44,6 +45,8 @@ var processEventConsumer *consumers.ProcessConsumer
 const processConsumerID = "gpu"
 const processConsumerChanSize = 100
 
+const defaultCollectedDebugEvents = 100
```

Comment:
please introduce a max bound as well to avoid some human mistakes that could lead to DOS